data/scripts.ts

```diff
@@ -572,11 +572,16 @@ export const Scripts: BattleScriptsData = {
 	tryMoveHit(target, pokemon, move) {
 		this.setActiveMove(move, pokemon, target);
 
-		if (!this.singleEvent('Try', move, null, pokemon, target, move)) {
+		let hitResult = this.singleEvent('Try', move, null, pokemon, target, move);
+		if (!hitResult) {
+			if (hitResult === false) {
+				this.add('-fail', pokemon);
+				this.attrLastMove('[still]');
+			}
```

Comment:
Ah right, this was added to the other call site to fix future moves (well it makes more sense than individually annotating each move). But at this point, is there any difference between the `Try` and `PrepareHit` singleEvents? (Aside from the parameter order of course.)

Reply:
There is some stuff after ``Try`` but before Protean, notably future attacks. Might not hurt to repurpose it, but that's beyond the scope of what I want to do in this PR.